setup.cfg

```diff
@@ -38,6 +38,10 @@ xfail_strict = true
 addopts = --strict-markers
 markers =
     backend
+    clickhouse
+    csv
+    dask
+    datafusion
```

Comment:
Shouldn't these be causing CI to fail right now if they're actually necessary?

Reply:
Those have been restored from a git merge, they were removed in #2903